docs/design/custom-tab-placement.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Position a custom tab on the ribbon
+description: 'Learn how to control where a custom tab appears on the Office ribbon and whether it has focus by default.'
+ms.date: 11/01/2020
+localization_priority: Normal
+---
+
+
+# Position a custom tab on the ribbon (preview)
+
+You can specify where you want your add-in's custom tab to appear on the Office application's ribbon by using markup in the add-in's manifest.
+
+> [!NOTE]
```

Comment:
You've included this note and the following IMPORTANT one a couple of times. Would it make sense to create an include file or two?

Reply:
I'm going to hold off on that for this PR because the PM might want the 2 articles where the Note is to be merged into a single article. The Important note actually has slightly variant test in at least one place where it appears (the 2nd bullet is not there), so using an include would be complicated